### DISCUSSION TITLE
Q: 在hexo-fluid博客上提交评论时弹出failed to fetch，该怎么解决？ || Q: When submitting a comment on the hexo-fluid blog, failed to fetch pops up. How to solve it?

### DISCUSSION THREAD

**Discussion by kxr189** [walinejs/waline#2177] [Q&A]
### Describe the bug | 问题描述

不知道该怎么解决了

### Website URL | 问题网站

https://ke-xirong.gitee.io/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Don't know how to solve it

### Website URL | Problem website

https://ke-xirong.gitee.io/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
https://github.com/walinejs/waline/issues/1318 vercel.app 域名国内访问有问题，需要绑定个域名，看下 issue

**Reply by kxr189** [reply]
管理端可以进入，只是提交评论会有问题也是需要绑定域名吗

---
The management side can be accessed, but there will be problems when submitting comments. Do I need to bind a domain name?

**Reply by zhoujiegl** [reply]
你看看你引入客户端的代码，有一句： serverURL: 'https://your-domain.vercel.app', 这里的链接要改为自己部署好的链接


---
Look at the code you introduced into the client. There is a sentence: serverURL: 'https://your-domain.vercel.app'. The link here should be changed to the link you deployed yourself.

**Reply by lanhui672** [reply]
我也是这样

这里在哪里看呢 -------引入客户端的代码，有一句： serverURL: 'https://your-domain.vercel.app/', 这里的链接要改为自己部署好的链接

**Comment by lanhui672** [comment]
我也是这样

这里在哪里看呢 -------引入客户端的代码，有一句： serverURL: 'https://your-domain.vercel.app/', 这里的链接要改为自己部署好的链接

**Comment by lanhui672** [comment]
我解决了，是服务ulr链接有问题。可以看我的教程，纠正了网上教程的错误。https://www.lanhui.live/post/2/

---
I solved it. There was a problem with the service ulr link. You can read my tutorial and correct the mistakes in the online tutorial. https://www.lanhui.live/post/2/

**Reply by SummerBeachhh** [reply]
无法访问

---
Unable to access

**Reply by SummerBeachhh** [reply]
好了，翻你仓库找到了，谢谢

---
Okay, I found it in your warehouse, thank you

**Reply by sunshang-hl** [reply]
![image](https://github.com/user-attachments/assets/d38cbc27-ece8-4155-b454-aa9644355992)  
serverURL 填Domains下的网址

**Reply by xiechimon** [reply]
非常感谢


---
Thank you so much
